QCamera2/QCamera2Factory.cpp

```diff
@@ -39,7 +39,7 @@
 #include "HAL/QCamera2HWI.h"
 #include "HAL3/QCamera3HWI.h"
 #include "QCamera2Factory.h"
-#include "QCameraMuxer.h"
+#include "HAL/QCameraMuxer.h"
```

Comment:
hardware/sony/camera/QCamera2/HAL/QCameraMem.cpp:37:29: fatal error: QComOMXMetadata.h: No such file or directory
 #include <QComOMXMetadata.h>

in aosp the path is hardware/qcom/camera...... not hardware/sony/camera